services/codeclimate/codeclimate-analysis.tester.js

```diff
@@ -77,6 +71,6 @@ t.create('malformed response for inner specific repo query')
   ) // No data.
   .networkOn() // Combined with allowUnmocked: true, this allows the outer user repos query to go through.
   .expectBadge({
-    label: 'maintainability',
-    message: 'invalid',
+    label: 'analysis',
+    message: 'invalid response data',
```

Comment:
It might be worth keeping these two tests. They did catch errors in the way I had things wired up.